plugins/@grouparoo/spec-helper/src/lib/cliSpecHelper.ts

```diff
@@ -247,6 +227,48 @@ DATABASE_URL="sqlite://grouparoo_test.sqlite"
     return runCliCommand(`generate group:manual ${groupName} --overwrite`);
   }
 
+  export async function generatePropertyToBootstrap(
+    runCliCommand: Function,
+    projectPath: string,
+    propertyName: string,
+    sourceId: string,
+    options?: CLIValidationOptions
+  ) {
+    await runCliCommand(
+      `generate manual:property ${propertyName} --parent ${sourceId}`
+    );
```

Comment:
Does this mean that you are saying a `manual:property` belongs to a `postgres:source:table`?  That doesn't seem right (and will fail as of https://www.pivotaltracker.com/story/show/177875103).   You might need to read the source file to learn it's type to know what kind of property to make from it.

Reply:
I don't think that's the case here because property templates are pretty much the same throughout all sources and don't have a `type` on them that's specific to the parent (source). Instead, the options between them may be different. 

Initially I solved this by passing an object with the options it should set on the bootstrapped property when calling `validateGenerators`, but after reading this I went with something a bit smarter and am now generating the right property for each generator.